src/IO/Resources/AnimationsLoader.cs

```diff
@@ -886,7 +886,7 @@ public void ConvertBodyIfNeeded(ref ushort graphic, bool isParent = false, bool
 
                 do
                 {
-                    if ((DataIndex[newGraphic].HasBodyConversion || !dataIndex.HasBodyConversion) && !(DataIndex[newGraphic].HasBodyConversion && dataIndex.HasBodyConversion))
```

Comment:
i think this will cause issues because iirc we need to be sure that `new` and `old` graphics don't have a conversion.
So the conversion will happen when:
new graphic index has a conversion OR the old one doesn't
AND
both doesn't have a conversion.

Reply:
Ok so if we define
A = DataIndex[newGraphic].HasBodyConversion
B = dataIndex.HasBodyConversion

Then this statement is
(A || !B) && !(A &&B)

If I then build a table of inputs (A, B):

(T, T) -> F
(T, F) -> T
(F, T) -> F
(F, F) -> T

So it looks like it's actually equivalent to !B. In other words, the condition is

if (! dataIndex.HasBodyConversion)

Does that look correct to you? I'll update you that if so.

Reply:
Im from my mobilephone so it's a little bit hard to review.
Yeah right, but in the commit i see:
`if (DataIndex[newGraphic].HasBodyConversion)` which should be wrong

Reply:
Yes I have it wrong in the patch. I'll update.